### DISCUSSION TITLE
Cannot import PhantomJS from Selenium?

### DISCUSSION THREAD

**Discussion by bethlet** [wkarney/minted-export#9] [Q&A]
Hi there! So excited about this script! What the heck, Minted?? I'm running Python 3.9.1. Thoughts about what could be causing this?

```
$ python minted_api_request.py
Traceback (most recent call last):
  File "/Users/bethlet/Projects/minted-export/minted_api_request.py", line 6, in <module>
    from seleniumrequests import Chrome
  File "/Users/bethlet/Projects/minted-export/env/lib/python3.9/site-packages/seleniumrequests/__init__.py", line 1, in <module>
    from selenium.webdriver import Firefox, Chrome, Ie, Opera, Safari, PhantomJS, Android, Remote
ImportError: cannot import name 'PhantomJS' from 'selenium.webdriver' (/Users/bethlet/Projects/minted-export/env/lib/python3.9/site-packages/selenium/webdriver/__init__.py)
```

**Comment by wkarney** [comment]
Hi @bethlet - thanks for giving the script a try - sorry you're running into issues though. The issue seems to originate with the `seleniumrequests` package - it's trying to import modules from `selenium.webdriver` that are no longer part of it, namely PhantomJS and Android. I'll refactor the code to fix these issues **asap**. 

In the meantime, there's another more manual approach that you can take to download your address book as a json file. Another user has laid it out in another [thread](#7). You login to minted, and then navigate to https://addressbook.minted.com/api/contacts/contacts/?format=json , and save that page source to your computer. Once you have the json file of your addresses you can convert it to an excel compatible format (csv, xlsx) using the script in that [thread](#7) (I'm in process of adding that workflow to the repo) or another online service such as https://json-csv.com.

Let me know how the export goes! I'm here if you run into further issues. 

Thanks!
-Will

**Comment by wkarney** [comment]
@bethlet I updated the code to remove the `seleniumrequests` dependency. Pull down a new version of the repo and give it a go!
